source/guide_mailman-3.rst

```diff
@@ -464,6 +464,100 @@ Install cronjobs
  @daily /home/$USER/.local/bin/mailman digests --send
 
 
+Configuration (almost) without qmail 
+====================================
+
+Since qmail will not be available within Uberspace U8, there is need for an alternative setup. There are multiple possibilities to achieve this, we will use ``fetchmail``  and a sieve filter here. This workflow is incompatible with a global ``~/.qmail-default``, but individual ``.qmail`` files for each list can happily coexist with this new approach. So you can test the new workflow without breaking existing lists.
+
+We start by adding a cachall mail box who will get all mail that does not belogn to any existing mailbox (its password will be called ``CATCHALLPWD``):
+
+.. code-block:: bash
+
+  uberspace mail user add catchallbox
+  uberspace mail catchall set catchallbox
+
+You may want to add a Sieve rule and an IMAP folder ``Mailman``. To do this use your preferred mail client or login via https://webmail.uberspace.de/ using ``catchallbox@isabell.uber.space`` and ``CATCHALLPWD`` as credentials. Under Settings > Filters you can add a custom sieve filter like this:
+
+.. code-block:: sieve
+
+  require["fileinto","regex","envelope"];
+
+  if anyof(
+    address :domain :regex ["to","delivered-to"] "lists.isabell.uber.space",
+    envelope :domain :regex "to" "lists.isabell.uber.space"
+  )
+  { 
+      fileinto "Mailman";
+      stop;
+  }
+
+A sieve filter is only useful if your list adresses differ from other adresses either because they have (a) specific domain(s) like ``lists`` or each mailing list contains a certain key word like ``list``. If all your mails are destined for Mailman 3 you do not need a filter or subfolder.
+
+Next, we add a ``fetchmailrc`` to, e.g., ``/home/isabell/mailman-suite/fetchmailrc`` (as there are already most other Mailman related config files). This file may look like this:
+
+.. code:: ini
+
+  #actually we use the idle option, but this has still effects in case of an error
+  set daemon 60
+  set no syslog
+  #has no effect with --nodetach, but ensures that we log to the same file, when we debug manually
+  set logfile /home/isabell/var/logs/fetchmail_out.log
+
```

Comment:
2 further options:
```
set no bouncemail
set postmaster isabell@uber.space
```
This sends error messages to another address and does not use the LMTP connection for that, because this is a global option and LMTP is server+user specific.

Reply:
> This assumes that mailing list has address mymailinglist@isabell.uber.space. At least the domain of the mailing list address must end in isabell.uber.space (subdomain.isabell.uber.space is also fine), otherwise the configuration option  localdomains isabell.uber.space must be adapted to contain (a suffix of) the domain. 

That was the problem I left it at isabell.uber.space. Changing it to lists.mydomain.tld resolved the error.

Thanks for your work on this! As a guide follower I couldn't have figured out on my own how to get rid of qmail files.

This also fixes https://github.com/Uberspace/lab/issues/1741